plugin/goto.py

```diff
@@ -63,18 +67,18 @@ def handle_response(self, response: Any) -> None:
             else:
                 locations = process_response_list(response)
             if len(locations) == 1:
-                open_location(window, locations[0])
+                open_location(window, locations[0], side_by_side)
             elif len(locations) > 1:
                 window.show_quick_panel(
                     items=locations,
-                    on_select=lambda x: select_entry(window, locations, x, view),
+                    on_select=lambda x: select_entry(window, locations, x, view, side_by_side),
```

Comment:
When quick panel is used the flow is maybe a bit weird. The files are previewed using full window view and on selecting a file it's opened in side-by-side view. I guess it's not that bad.